src/Components/Wizard/steps/ReviewDetails/index.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { Form, FormGroup, Title } from '@patternfly/react-core';
+import { ExpandableAWS } from '../../../ExpandableAWS';
+
+const ReviewDetails = () => {
+  return (
+    <Form isWidthLimited maxWidth="80%">
```

Comment:
do we need to wrap the review step with a `<Form>`? basically, it's a read-only review with no inputs. no?

Reply:
I tend to agree here :thinking:

Reply:
yes, I agree, so what is it supposed to be? 
Text Input? 
I need to warp the ExpandableAWS and the two titles somehow

Reply:
Why do you need to wrap? wouldn't empty component `<></>` be enough there?

Reply:
I changed it to Stack and each component inside is inside a Stack item. 
I also updated the screenshot